Formula/php.rb

```diff
@@ -289,6 +290,8 @@ def plist; <<~EOS
     system "#{sbin}/php-fpm", "-t"
     system "#{bin}/phpdbg", "-V"
     system "#{bin}/php-cgi", "-m"
+    system "#{bin}/pecl", "install", "xdebug"
```

Comment:
The sandbox isn't going to let you write here.

Reply:
Yeah I noticed that. I thought I could get around it by using `config-set extension_dir` but it didn't seem to work when I tried it. Haven't looked at the pecl code but I'm guess the settings comes from phpize/php-config.